pages/linux/uuidparse.md

```diff
@@ -0,0 +1,28 @@
+# uuidparse
+
+> Parse universally unique identifiers.
+> More information: <https://manned.org/uuidparse.1>.
+
+- Parse the specified UUIDs, use a tabular output format:
+
+`uuidparse {{uuid1 uuid2 ...}}`
+
+- Parse UUIDs from `stdin`:
+
+`{{command}} | uuidparse`
+
+- Use the JSON output format:
+
+`uuidparse --json {{uuid1 uuid2 ...}}`
+
+- Do not print a header line:
+
+`uuidparse --noheadings {{uuid1 uuid2 ...}}`
+
+- Use the raw output format:
+
+`uuidparse --raw {{uuid1 uuid2 ...}}`
```

Comment:
```suggestion
`uuidparse --raw {{uuid1 uuid2 ...}}`

- Specify which of the 4 output columns to print:

`uuidparse --output {{UUID,VARIANT|TYPE,TIME}}`
```
I think it would be nice to include the output example. Do you think this is a great way to show all possibilities?